lib/sync-settings.coffee

```diff
@@ -31,6 +31,9 @@ module.exports =
     atom.commands.add 'atom-workspace', "sync-settings:backup", => @backup()
     atom.commands.add 'atom-workspace', "sync-settings:restore", => @restore()
 
+    # Clean up after #36, see #68 for more info.
+    fs.unlink(require('path').join(atom.getConfigDirPath(), 'snippets.coffee'), ->)
```

Comment:
Since the original bug was wrong filename, i suggest to compare the `snippets.coffee` and `snippets.cson` and delete only if they match or the `snippets.coffee` is empty (contain the empty comment line that is)